web/src/popover_menus_data.js

```diff
@@ -118,3 +127,26 @@ export function get_actions_popover_content_context(message_id) {
         should_display_quote_and_reply,
     };
 }
+
+export function get_topic_popover_content_context(elt) {
```

Comment:
The goal is for this module to contain pure functions that don't interact with the DOM, that can have automated tests. 

So I think you want to keep the jQuery/DOM parsing in the caller, and have this function just be passed a `(stream_id, topic_name, url)` or similar.

Reply:
This will also help avoid an import loop; this is a data module and shouldn't import a UI/jQuery manipulation module like `stream_povover.js`

Reply:
The code has been refactored so that `popover_menus_data` only includes function calls. All DOM/JQuery parsing is now performed in `popover_menus` before calling the `popover_menus_data` file.

Reply:
You removed the jQuery import, but not the `strema_popover` module -- read more carefully next time :).

I went ahead and fixed this just so I can integrate it and avoid another review round trip.

Reply:
Sorry for the inconvenience. I will make sure to double-check from next time.